util/tlgen/validate.py

```diff
@@ -121,7 +121,7 @@
 # by inspecting the base addresses. Note that the validation
 # script also ensures that base addresses are aligned with
 # to this granularity.
-MIN_DEVICE_SPACING = 0x1000
+MIN_DEVICE_SPACING = 0x100
```

Comment:
> I don't foresee a problem with reducing this to 256 B (i.e., the new `0x100` value), as TL-UL doesn't support bursts (which could cross devices). When a range of addresses is written, e.g., through a sequence of stores by the CPU or of writes by the DMA, SW/FW is responsible for adhering to the system address map and boundaries between devices.

That's true. That must also be considered when writing on 4k page sizes.

If you want to restrict that, you could make this as a parameter of the crossbar definition in the top-level hjson file.

Reply:
> you could make this as a parameter of the crossbar definition in the top-level hjson file.

Interesting suggestion, would this be feasible as part of this PR?

Reply:
Since this PR is blocking another PR, I would like to do that as a follow-up PR if you don't mind. The parameter is not causing any RTL impact, thus I would like to leave that PR as it is.

Reply:
OK agreed

Reply:
Tracked in #25153.